src/src.builds

```diff
@@ -1,8 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
-  <ItemGroup Condition="'$(OS)' != 'Windows_NT'">
-    <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Memory/src/System.Memory.csproj" />
+  <ItemGroup>
+    <ProjectExclusions Condition="'$(OS)' != 'Windows_NT'" Include="$(MSBuildThisFileDirectory)**\src\System.Memory.csproj" />
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)**\src\System.Threading.Overlapped.csproj" />
```

Comment:
Lets make sure we have an issue to track fixing these projects and removing them from this list and the package ignore list.